Godeps/_workspace/src/k8s.io/kubernetes/pkg/master/master.go

```diff
@@ -168,6 +168,11 @@ type Config struct {
 	SSHUser       string
 	SSHKeyfile    string
 	InstallSSHKey InstallSSHKey
+
+	// Used to make the service error resolver aware of apis that may be installed in addition
+	// to the kubernetes api so it can parse request info appropriately and show more detailed
+	// error messages.
+	ExtendedApiPrefixes []string
```

Comment:
doesn't the API server already something like this in order to support the experimental API group?